ipv8/REST/json_util.py

```diff
@@ -122,3 +137,25 @@ def load(fp, *args, **kwargs):
     :return: the Python object(s) extracted from the JSON input.
     """
     return json.load(fp, *args, **kwargs)
+
+
+def ensure_serializable(value):
```

Comment:
This can become **very expensive** if we do this on every dump call. This rewards bad programming and slows down well written code. Please just error out like before: when a programmer feeds in a value which is non-json.

Reply:
Yes, `ensure_serializable` can be an expensive operation but with `decode_bytes=False` set by default should ensure this serializable assurance only happens when it is specifically set to True.